app/src/pages/settings/APIKeysTable.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo } from "react";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+
+import { Flex, Icon, Icons } from "@arizeai/components";
+
+import { TextCell } from "@phoenix/components/table";
+import { tableCSS } from "@phoenix/components/table/styles";
+import { TableEmpty } from "@phoenix/components/table/TableEmpty";
+import { TimestampCell } from "@phoenix/components/table/TimestampCell";
+
+import { DeleteAPIKeyButton } from "./DeleteAPIKeyButton";
+import { APIKey } from "./types";
+export function APIKeysTable({
+  data,
+  handleDelete,
+}: {
+  data: APIKey[];
+  handleDelete: (id: string) => void;
+}) {
+  const columns = useMemo(() => {
```

Comment:
this is unchanged just moved the table logic out here to be shared be tween user and system tables